### PR TITLE
#248  Move npm-install from dc.yml to Dockerfile

### LINKED ISSUE #248: Refactor nodejs static building
Обязательна для #269 

-----

Задачу создал из этого коммента
https://github.com/fidals/shopelectro/pull/246#discussion_r158480687

Сейчас у нас проблема в том, что `make static` зависит от `make build`. Это создаёт сложности.
Да и к билду статики через ноду давненько есть вопросы (у Дюка точно).
Пересмотри билд статики и сделай так, чтобы `make build` среди прочего билдил и статику.

Итого критерий завершения задачи:
`make build` билдит статику и код архитектурно хорош

### PR DESCRIPTION
**Pull request by duker33**
Closes #248 

- Fix drone. Improve makefile
- Drone now builds and pushes fidals/se-nodejs image
- Move npm-install from dc.yml to Dockerfile

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -60,11 +60,21 @@ pipeline:
...
+    when:
+      status: [ success, failure ]
+
   docker-build:
     image: docker/compose:1.17.1
     commands:
       - cd docker
-      - docker-compose -f docker-compose.yml build --no-cache se-python
+      - docker-compose build --no-cache se-nodejs
+      - docker-compose build --no-cache se-python
```
Comment (artemiy312) [inline .drone.yml]: We may build few images inline `docker-compose build --no-cache se-nodejs se-python`

File: docker/Makefile [hunk 2]
```diff
@@ -38,6 +39,7 @@ lint:
 
 # start dev environment
 dev: build
```
Comment (artemiy312) [inline docker/Makefile]: We should not run `build` before `dev`, isn't it?

Instead it should be:
```Makefile
dev:
    $(dc) pull se-python
    $(MAKE) static
    ...
```

**Comment by artemiy312** [comment]
@duker33 I have finished review for you

File: .drone.yml (hunk 1 above)
Comment (duker33) [reply .drone.yml]: TODO

**Comment by duker33** [comment]
CI completed, but github can't get it's status. Merge it

**Pull request merged**